frontend_vue/src/components/base/BaseSpinner.vue

```diff
@@ -32,6 +33,8 @@ const spinnerClass = computed(() => {
       return `border-green-500`;
     case SpinnerVariantEnum.SECONDARY:
       return 'border-white';
+    case SpinnerVariantEnum.DANGER:
```

Comment:
are we using this anywhere?

Reply:
It's needed for the loading spinner on the DANGER button

Reply:
I'm not sure I get it

`const spinnerVariant = computed(() => {
  switch (props.variant) {
    case ButtonVariantEnum.PRIMARY:
      return ButtonVariantEnum.SECONDARY;
    case ButtonVariantEnum.DANGER:
      return ButtonVariantEnum.SECONDARY
    default:
      return ButtonVariantEnum.PRIMARY
  }
})`

here we never pass the variant danger to the spinner, right? 
The button Danger adds the variant Secondary to the spinner
Or am I missing something?

Reply:
I've updated